src/libData/AccountStore/AccountStoreSCEvm.cpp

```diff
@@ -64,30 +126,73 @@ void AccountStoreSC<MAP>::EvmCallRunner(const INVOKE_TYPE /*invoke_type*/,  //
   switch (fut.wait_for(std::chrono::seconds(EVM_RPC_TIMEOUT_SECONDS))) {
     case std::future_status::ready: {
       LOG_GENERAL(WARNING, "lock released normally");
-      if (zil::metrics::Filter::GetInstance().Enabled(
-              zil::metrics::FilterClass::ACCOUNTSTORE_EVM)) {
-        m_accStoreProcees->Add(1, {{"lock", "release-normal"}});
+
+      INCREMENT_CALLS_COUNTER(GetInvocationsCounter(), ACCOUNTSTORE_EVM, "lock",
+                              "release-normal");
+      if (TRACE_ENABLED(ACC_EVM))
+        span->AddEvent("return", {{"reason", "release-normal"}});
+
+      // Example code for pushing a context into a span and then creating a new
+      // span from the context carried
+      if (TRACE_ENABLED(ACC_EVM))
+      {  // example for Artem - retreving the context for shipment over the
+         // wire.
+        auto current_ctx = context::RuntimeContext::GetCurrent();
+        ArtemTextMapCarrier<std::map<std::string,std::string>> carrier;
+        auto prop = context::propagation::GlobalTextMapPropagator::
+            GetGlobalPropagator();
+        prop->Inject(carrier, current_ctx);
+
+        for (auto ting : carrier.data_) {
+          std::cout << "key :" << ting.first << " ";
```

Comment:
is this production code? Looks like debug

Reply:
will be removed